mkosi/manifest.py

```diff
@@ -197,6 +199,31 @@ def record_deb_packages(self, root: Path) -> None:
 
             source_package.add(package)
 
+    def record_pkg_packages(self, root: Path) -> None:
+        package_descriptions = [str(p) for p in (root / "/var/lib/pacman/local").glob("*/desc")]
```

Comment:
Yes, with the `/` before `var` this ignores the initial `root`.

Reply:
Yes, but note this comment is done on an outdated revision :)

I did notice this error myself while testing.

Reply:
Yeah, I noticed I had some outdated comment when I was "publishing" the review. Github shows e.g. "2 comments", but if the comment is on a part of code that has been since changed, it doesn't show the comment anywhere. In the past it used to show all unpublished comments on the "comments" pane, but now it doesn't. So in the end I just pressed the green button without and forgot about this…